src/FixedDataTable.js

```diff
@@ -456,11 +456,18 @@ class FixedDataTable extends React.Component {
   }
 
   componentWillUnmount() {
+    // TODO (pradeep): Remove these and pass to our table component directly after
```

Comment:
Here's the related issue if you want to reference it:
https://github.com/facebook/react/issues/6436

Reply:
thanks, will put.